libmcount/plthook.c

```diff
@@ -817,9 +819,21 @@ unsigned long plthook_entry(unsigned long *ret_addr, unsigned long child_idx,
 	return real_addr;
 }
 
+unsigned long plthook_entry(unsigned long *ret_addr, unsigned long child_idx,
+			    unsigned long module_id, struct mcount_regs *regs)
+{
+	int saved_errno = errno;
+	unsigned long ret;
+	ret = __plthook_entry(ret_addr, child_idx, module_id, regs);
```

Comment:
It doesn't look consistent with other code. What about this if you want to avoid 80 chars in a line?
```c
unsigned long plthook_entry(unsigned long *ret_addr, unsigned long child_idx,
                            unsigned long module_id, struct mcount_regs *regs)
{
        int saved_errno = errno;
        unsigned long ret =
                __plthook_entry(ret_addr, child_idx, module_id, regs);

        errno = saved_errno;
        return ret;
}
```
Otherwise, there should be a empty line after declarations.
```c
unsigned long plthook_entry(unsigned long *ret_addr, unsigned long child_idx,
                            unsigned long module_id, struct mcount_regs *regs)
{
        int saved_errno = errno;
        unsigned long ret;

        __plthook_entry(ret_addr, child_idx, module_id, regs);
        errno = saved_errno;
        return ret;
}
```